news/_posts/2021-02-22-Singular.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Singular.jl 0.4.5 released
+author: Dan Schultz   
+---
+Here is an overview of the changes in this release of [Singular.jl](https://github.com/oscar-system/Singular.jl):
+
+- Singular.jl should no longer crash on the new rings that Singular creates, or at least it should crash in far fewer cases.
+- Singular's algebraic extensions of `QQ` and `Fp` can now be constructed via the
+function `AlgebraicFieldExtension`. This function takes a univariate `FunctionField`
+and a minimal polynomial in the function field to create the algebraic extension.
+- Various small bugs have been fixed.
+
+
+On behalf of the Singular.jl developers,
+
+Dan
```

Comment:
This is highly subjective, but I would omit this, seems somewhat redundant. Alas, I am not objecting, this is just my two cents, feel free to ignore :-)
```suggestion
```

Reply:
It would also be easy to add the author's name next to the date, like is shown at https://oscar.computeralgebra.de/news/, as otherwise the author's name is not visible in the post's page without this signature.